src/docs/get-started/codelab-web.md

```diff
@@ -262,6 +353,7 @@ Click it to bring up the welcome screen.
 Note how it animates in from the bottom.
 You get that behavior for free.
 </li>
+
 </ol>
 
 ### Problems?
```

Comment:
Correct me if I'm wrong, but now that we have the DartPad at the end with the complete example, why not just delete the three links to the gist files?

Reply:
I decided to remove the problems sections altogether. WDYT?